wasm/jit/jit_amd64.go

```diff
@@ -807,27 +910,32 @@ func (b *amd64Builder) handleConstI64(o *wazeroir.OperationConstI64) error {
 	return nil
 }
 
-func (b *amd64Builder) moveStackToRegister(tp generalPurposeRegisterType, loc *valueLocation) error {
+func (b *amd64Builder) moveStackToRegisterWithAllocation(tp generalPurposeRegisterType, loc *valueLocation) error {
```

Comment:
this and `releaseRegisterToStack` are the same rename thingy from #71 for clarity!